src/main/java/org/candlepin/subscriptions/tally/MetricUsageCollector.java

```diff
@@ -87,6 +87,12 @@ public CollectionResult collect(String serviceType, String accountNumber, DateRa
             .findById(new AccountServiceInventoryId(accountNumber, serviceType))
             .orElse(new AccountServiceInventory(accountNumber, serviceType));
 
+    // SWATCH-261 This logic should be implemented much cleaner
+    accountServiceInventory.getServiceInstances().values().stream()
```

Comment:
Why'd this jump locations out of curiosity?  Are we referencing the orgId in some of the method calls below or something?

Reply:
I moved it , so it easier to find in the clean up in the other cards, and I was using it but I came to the decision to just use the one liner